GitHubProjectColumns.ps1

```diff
@@ -0,0 +1,381 @@
+        [int64] $Column,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $telemetryProperties = @{}
+
+    $uriFragment = [String]::Empty
+    $description = [String]::Empty
+
+    if ($PSCmdlet.ParameterSetName -eq 'Project')
+    {
+        $telemetryProperties['Project'] = Get-PiiSafeString -PlainText $Project
+
+        $uriFragment = "/projects/$Project/columns"
+        $description = "Getting project columns for $Project"
+    }
+    elseif ($PSCmdlet.ParameterSetName -eq 'Column')
+    {
+        $telemetryProperties['Column'] = Get-PiiSafeString -PlainText $Column
+
+        $uriFragment = "/projects/columns/$Column"
+        $description = "Getting project column $Column"
+    }
+
+    $params = @{
+        'UriFragment' = $uriFragment
+        'Description' = $description
+        'AccessToken' = $AccessToken
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+        'AcceptHeader' = 'application/vnd.github.inertia-preview+json'
+    }
+
+    return Invoke-GHRestMethodMultipleResult @params
+}
+
+function New-GitHubProjectColumn
+{
+<#
+    .DESCRIPTION
+        Creates a new column for a Github project.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER Project
+        Id of the project to create a column for.
+
+    .PARAMETER Name
+        The name of the column to create.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no command line status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        New-GitHubProjectColumn -Project 999999 -Name 'Done'
+
+        Creates a column called 'Done' for the project with id 999999.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess)]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification = "Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+
+        [Parameter(Mandatory)]
+        [int64] $Project,
+
+        [Parameter(Mandatory)]
+        [string] $Name,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $telemetryProperties = @{}
+    $telemetryProperties['Name'] = Get-PiiSafeString -PlainText $Name
```

Comment:
We don't get any value with having this.  Names aren't unique and so this doesn't provide any meaningful data.  We could capture the the `PiiSafeString` of Project, and then we could identify how many times a specific project created columns with this module, but knowing the name doesn't really get us much.